src/main/java/com/guciowons/shoppingbasket/Basket/BasketController.java

```diff
@@ -0,0 +1,29 @@
+package com.guciowons.shoppingbasket.Basket;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/baskets")
+public class BasketController {
+    private final BasketService basketService;
+
+    public BasketController(BasketService basketService) {
+        this.basketService = basketService;
+    }
+
+    @RequestMapping(value = "/{id}/{quantity}", method = RequestMethod.PUT)
+    public ResponseEntity<String> addProductToBasket(@PathVariable int id, @PathVariable int quantity){
+        return basketService.addProductToBasket(id, quantity);
+    }
+
+    @RequestMapping(value = "/{id}/{quantity}", method = RequestMethod.DELETE)
+    public ResponseEntity<String> removeProductFromBasket(@PathVariable int id, @PathVariable int quantity){
+        return basketService.removeProductFromBasket(id, quantity);
+    }
+
+    @RequestMapping(method = RequestMethod.GET, produces = "application/json")
+    public BasketSummarized summarizeBasket(){
```

Comment:
Skoro wyżej zwracasz `ResponseEntity<String>` to tutaj też powinieneś zwrócić `ResponseEntity<BasketSummarized>`, żeby być spójnym -- wydaje mi się, że wtedy też nie trzeba ustawić `produces` bo to spring sam ogarnie